pkgs/os-specific/linux/open-iscsi/default.nix

```
@@ -1,36 +1,38 @@
-{ stdenv, fetchFromGitHub, automake, autoconf, libtool, gettext, utillinux, openisns, openssl, kmod }:
+{ stdenv, fetchFromGitHub, automake, autoconf, libtool, gettext,
+utillinux, openisns, openssl, kmod, perl, systemd, pkgconf }:
+
 stdenv.mkDerivation rec {
   name = "open-iscsi-${version}";
-  version = "2.0-873-${stdenv.lib.substring 0 7 src.rev}";
+  version = "2.0-877";
```

The versions actually have dots not dashes.
```suggestion
  version = "2.0.877";
```